.github/workflows/backport-to-release-branch.yml

```diff
@@ -0,0 +1,196 @@
+# This workflow moves commits from a development branch to a release branch.
+#
+# Each commit in the development branch is cherry-picked to the release branch. If a merge conflict occurs, the
+# commit is committed to a new branch with merge markers and then a PR is created into the target branch with those
+# markers. The PR is labeled with "type:release-merge-conflict" to indicate that it needs manual resolution.
+#
+# The PR is expected to fail compilation and status checks of course due to the merge conflict markers. A human should
+# then checkout the PR branch, resolve the conflicts, and push the changes back to the PR branch.
+#
+# This workflow is currently being tested in the Mu Basecore repository. Once it performs well over a reasonable
+# period of time, it will be moved to the Mu DevOps repository for general use and synced to other relevant
+# repositories using a more generalized template to fill in some of the details.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+name: Backport Commits to Release Branch
+
+on:
+  push:
+    branches:
+      - dev/202405

```

Comment:
The plan is to update this at a future point for n and n-1, based on version.njk?

Reply:
Yes. This it what's meant by this comment in the PR description: "Once it performs well over a reasonable period of time, it will be moved to the Mu DevOps repository for general use and synced to other relevant repositories using a more generalized template to fill in some of the details."

It is directly added to Mu Basecore in this PR to gather more usage data. When it is added to Mu DevOps and synced, some details like the branch will be filled in there.

Reply:
Definitely agreed on testing this out first, but when we go to mu_devops with this, we may want to update the checkbox to list the supported release branches (as defined in mu_devops) and allow the PR submitter to mark what branches it gets backported to? It may not be relevant to some release branches.